data/publications/index.yaml

```
@@ -1,3 +1,14 @@
+- title: Development and Benchmarking of Open Force Field 2.0.0 — the Sage Small Molecule Force Field
+  authors: "Simon Boothroyd, Pavan Kumar Behara, Owen Madin, David Hahn, Hyesu Jang, Vytautas Gapsys, Jeffrey Wagner, Joshua Horton, David Dotson, Matthew Thompson, Jessica Maat, Trevor Gokey, Lee-Ping Wang, Daniel Cole, Michael Gilson, John Chodera, Christopher Bayly, Michael Shirts, David Mobley"
+  summary: We introduce the Open Force Field (OpenFF)~2.0.0 small molecule force field for drug-like molecules, code-named Sage, which builds upon our previous iteration, Parsley. OpenFF force fields are based on direct chemical perception, which generalizes easily to highly diverse sets of chemistries based on substructure queries. Like the previous OpenFF iterations, the Sage generation of OpenFF force fields was validated in protein-ligand simulations to be compatible with AMBER biopolymer force fields. In this paper we detail the methodology used to develop this force field, as well as the innovations and improvements introduced since the release of Parsley 1.0.0. One particularly significant feature of Sage is a set of improved Lennard-Jones (LJ) parameters retrained against condensed phase mixture data, the first refit of LJ parameters in the OpenFF small molecule force field line. Sage also includes valence parameters refit to a larger database of quantum chemical calculations than previous versions, as well as improvements in how this fitting is performed. Force field benchmarks show improvements in general metrics of performance against quantum chemistry reference data such as root mean square deviations (RMSD) of optimized conformer geometries, torsion fingerprint deviations (TFD), and improved relative conformer energetics (ΔΔ𝐸). We present a variety of benchmarks for these metrics against our previous force fields as well as in some cases other small molecule biomolecular force fields. Sage also demonstrates improved performance in estimating physical properties, including comparison against experimental data from various thermodynamic databases for small molecule properties such as Δ𝐻_𝑚𝑖𝑥, ρ(𝑥), Δ𝐺_𝑠𝑜𝑙𝑣 and Δ𝐺_𝑡𝑟𝑎𝑛𝑠. Additionally, we benchmarked against protein-ligand binding free energies (Δ𝐺_𝑏𝑖𝑛𝑑), where Sage yields results statistically similar to previous force fields. All the data is made publicly available along with complete details on how to reproduce the training results at https://github.com/openforcefield/openff-sage.
+  img: noun_sage_1576600.png
+  date: 2022-11-21
+  preprint: 
+    server: chemRxiv
+    url: https://doi.org/10.26434/chemrxiv-2022-n2z1c
+    licence: "CC BY 4.0"
+    date: 2022-11-21
+  code: https://github.com/openforcefield/openff-sage
```

The `code:` formatting here automatically adds the `https://github.com/` bit to the URL. So the original form of this link ends up having it twice and being https://github.com/https://github.com/openforcefield/openff-sage. Be sure to render the website locally and test the links in the future to catch this sort of thing :-) 

```suggestion
  code: openforcefield/openff-sage
```